### PR TITLE
lesson 1.2

### PR DESCRIPTION
**Pull request by kmg37**
refactored App.js and added form component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,13 @@
 import React from "react";
-
-const todoList = [
-  { title: "This is a title", id: 1 },
-  { title: "Yet another title", id: 2 },
-  { title: "The last title", id: 3 },
-];
+import TodoList from "./todoList";
```
Comment (mattcarlotta) [inline src/App.js]: Requires change:
This import should be importing a [TodoList.js (1st requirement)](https://github.com/Code-the-Dream-School/react/wiki/Lesson-1.2#move-list-into-new-todo-list-component) file. The reason you ran into the VSCode error is that the file is currently named as `todoList.js` and imports are case sensitive!

**Review by mattcarlotta — CHANGES_REQUESTED** [review]
One minor mistake with the naming of a file; otherwise, the assignment is successful! Please fix the item below and then "@" me in a feedback comment to let me know the PR is ready for another review.

File: src/App.js (hunk 1 above)
Comment (kmg37) [reply src/App.js]: @mattcarlotta fixed! Thank you for the feedback. Also, forgot to ask, but should I be making a new branch for each lesson?

Comment (mattcarlotta) [reply src/App.js]: @kmg37 Yes! This assignment should be in a `lesson_1_2` branch.

Comment (mattcarlotta) [reply src/App.js]: @kmg37 You'll also have to rename the `todoList.js` file to `TodoList.js`. As mentioned in my Slack DM, you may have done it, but it's not being reflected in the PR.

**Pull request closed without merging**